src/metrics.rs

```diff
@@ -2232,10 +2241,18 @@ impl GooseMetrics {
         if let Some(starting) = self.starting {
             let second = (Utc::now().timestamp() - starting.timestamp()) as usize;
 
-            expand_per_second_metric_array(&mut self.users_per_second, second, 0);
+            let last_user_count = match self.users_per_second.last() {
+                Some(last) => *last,
+                None => 0,
+            };
+            expand_per_second_metric_array(&mut self.users_per_second, second, last_user_count);
             self.users_per_second[second] = self.users;
 
-            expand_per_second_metric_array(&mut self.tasks_per_second, second, 0);
+            let last_task_count = match self.tasks_per_second.last() {
+                Some(last) => *last,
+                None => 0,
+            };
+            expand_per_second_metric_array(&mut self.tasks_per_second, second, last_task_count);
```

Comment:
Users and tasks metric are not tied to a specific request so we still need to rely on `Utc:now()`. But this is OK since we will hit this immediately after a new user was spawned so the data should be accurate. The only thing we need to make sure is to carry the last count over the seconds we missed while the thread was sleeping to avoid gaps on the graph.

Reply:
Tasks metrics has `thread_user.started.elapsed().as_millis(),` which is what is in the logs and is already accurate, why can't we use that?

Reply:
And for users, why can't we also use `thread_user.started`?

Reply:
I used `GooseTaskMetric.elapsed`, which also required me to move this code into `receive_metric()` again. But this time it is in the place where task metrics are received, so I think that it makes more sense.